src/charm.py

```diff
@@ -58,6 +78,8 @@ def CinderThreeParContext(charm_config, service):
         ctxt.append((
             'volume_driver',
             'cinder.volume.drivers.hpe.hpe_3par_iscsi.HPE3PARISCSIDriver'))
+    else:
+        raise InvalidConfigError('Invalid config (driver-type)')
```

Comment:
LGTM now, thanks. We should document this in the docstring with `:raises: InvalidConfigError`.